scripts/import-ai-lab-samples

```diff
@@ -57,6 +61,9 @@ for f in */; do
         sed -i "s!sed.edit.APPTAGS!$APP_TAGS!g" $DEST/template.yaml
         sed -i "s!sed.edit.CATALOG_DESCRIPTION!Secure Supply Chain Example for $APP_DISPLAY_NAME!g" $DEST/template.yaml
 
+        sed -i "s!sed.edit.NAME!$APP_NAME!g" $DEST/docs/index.md
```

Comment:
```suggestion
        sed -i "s!sed.edit.NAME!$APP_DISPLAY_NAME!g" $DEST/docs/index.md
```

Consider using Display Name, it seems Name is mostly used for id with it being hyphenated